data/records/alice-2010-pbpb.json

```diff
@@ -0,0 +1,69 @@
+[
+  {
+    "abstract": {
+      "description": "Lead-Lead data sample at the collision energy of 2.76TeV from run number 137135 of the LHC10h data taking period."
+    },
+    "accelerator": "CERN-LHC",
+    "collaboration": {
+      "name": "ALICE Collaboration"
+    },
+    "collections": [
+      "ALICE-Reconstructed-Data"
+    ],
+    "collision_information": {
+      "energy": "2.76TeV",
+      "type": "PbPb"
+    },
+    "created": "2025-02-11T15:53:44.819508",
+    "date_created": [
+      "2010"
+    ],
+    "date_published": "2025",
+    "distribution": {
+      "formats": [
+        "root"
+      ],
+      "number_events": 213750,
+      "number_files": 128,
+      "size": 2641170373
+    },
+    "experiment": [
+      "ALICE"
+    ],
+    "files": [
+      {
+        "checksum": "adler32:7ac174b9",
+        "size": 27508,
+        "type": "index.json",
+        "uri": "root://eospublic.cern.ch//eos/opendata/alice/upload/2010/LHC10h/000139510/file-indexes/LHC10h_000139510_PbPb_2.76TeV_205656_file_index.json"
+      }
+    ],
+    "license": {
+      "attribution": "CC0-1.0"
+    },
+    "publisher": "CERN Open Data Portal",
+    "recid": "1114",
+    "title": "LHC10h_000137135_PbPb_2.76TeV_213750",
```

Comment:
Note that this does not look consistent with the old existing titles such as "LHC2010h_PbPb_ESD_138275". Shall we harmonise them?

Reply:
I was pointed to such a name: `{PERIOD}_{RUN}_{BEAMTYPE}_{ENERGY}_{COLLISIONS_NR}`
This data is converted data to O2 ALICE framework, there are no longer ESD/AOD present, only AO2D.root files. Let me know if different name is required and with what content.